migration/main.py

```diff
@@ -1,45 +1,55 @@
 #!/usr/bin/env python3
 import getopt
 import sys
+import io
 import lpass
 from folder_migrate import migrate_folders
 from vault_item_import import migrate_items
 
 
 def main(argv):
-    csv_data = ''
+    options = {
+        'ignore-shared': False,
+    }
+    csvfile = None
     is_migrating_folders = False
     is_migrating_items = False
-    opts, args = getopt.getopt(argv, "fi", ["file=", "folders", "items"])
+    opts, args = getopt.getopt(argv, "di", ["file=", "directory", "item", "ignore-shared"])
     for opt, arg in opts:
         if opt == "--file":
             print(f'Export secrets from csv file {arg}')
-            with open(arg, newline='') as csvfile:
-                csv_data = lpass.prepare_csv(csvfile.read())
+            csvfile = open(arg, newline='')
             continue
 
-        if opt in ("-f", "--folders"):
+        if opt in ("-d", "--directory"):
             is_migrating_folders = True
             continue
 
-        if opt in ("-i", "--items"):
+        if opt in ("-i", "--item"):
             is_migrating_items = True
             continue
 
```

Comment:
If someone chooses `-id` or `--item --directory`, we should proceed as if they selected just `-i` since `-i` and `-id` are equivalent but `-id` fails with an uninformative error.

Reply:
As peer our discussion, we will force user to set only one flag per run either `-i` or `-d` .